swan-lake/integration-tools/edi-tool.md

```diff
@@ -61,339 +61,238 @@ The command options that are available with the tool are listed below.
+ballerina/editoolspackage:0.8.5 pulled from central successfully
+tool 'edi:0.8.5' pulled successfully.
+tool 'edi:0.8.5' successfully set as the active version.
 ```
 
-## Example
+## Examples
 
 Examples of the above usages are described below.
 
 ### Code generation example
 
 Follow the steps below to try out an example of generating Ballerina code from an EDI schema file.
 
-1. Create a sample EDI schema (`edi-schema.json` file) with the code below.
+#### Clone the example project
 
-    >**Info:** The below schema can be used to parse EDI documents with one HDR segment (mapped to `header`) and any number of ITM segments (mapped to `items`). The HDR segment contains three fields, which are mapped to `orderId`, `organization`, and `date`. Each ITM segment contains two fields mapped to `item` and `quantity`.
+Clone the [artifacts of the example](https://github.com/ballerina-guides/integration-samples/tree/main/edi_code_generation) and extract them to a preferred location.
 
-    ```json
-    {
-        "name": "SimpleOrder",
-        "delimiters" : {"segment" : "~", "field" : "*", "component": ":", "repetition": "^"},
-        "segments" : [
-            {
-                "code": "HDR",
-                "tag" : "header",
-                "fields" : [{"tag": "code", "required": true}, {"tag" : "orderId"}, {"tag" : "organization"}, {"tag" : "date"}]
-            },
-            {
-                "code": "ITM",
-                "tag" : "items",
-                "maxOccurances" : -1,
-                "fields" : [{"tag": "code", "required": true}, {"tag" : "item"}, {"tag" : "quantity", "dataType" : "int"}]
-            }
-        ]
-    }
-    ```   
-
-2. Generate Ballerina records for the above EDI schema.
-
-    >**Info:** The generated Ballerina records will be saved in a file named `orderRecords.bal`.
+>**Info:** The cloned directory includes the artifacts that will be required to try out this example. The `schemas` folder includes the schema file ([`schemas/edi-schema.json`](https://github.com/ballerina-guides/integration-samples/tree/main/edi_code_generation/schemas/edi-schema.json)), which can be used to parse EDI documents with one HDR segment (mapped to `header`) and any number of ITM segments (mapped to `items`). The HDR segment contains three fields, which are mapped to `orderId`, `organization`, and `date`. Each ITM segment contains two fields mapped to `item` and `quantity`. The [`main.bal`](https://github.com/ballerina-guides/integration-samples/tree/main/edi_code_generation/main.bal) file includes the business logic/usage of the code, and the [`resources/edi-sample.edi`](https://github.com/ballerina-guides/integration-samples/tree/main/edi_code_generation/resources/edi-sample.edi) file includes the EDI message to be read from the generated code.
+
+#### Generate the code
+
+Follow the steps below to generate Ballerina records for the above EDI schema.
+
+1. Navigate to the cloned `edi_code_generation` directory.
```

Comment:
Avoid underscores for directory names, Fix the samples repo too. Use hyphens as it will be more readable in URLs etc.